policyengine_us/variables/gov/ed/pell_grant/pell_grant.py

```diff
@@ -17,3 +20,20 @@ def formula(person, period, parameters):
         capped = min_(unbounded, p.amount.max)
         amount = where(capped < p.amount.min, 0, capped)
         return amount * (months_in_school / p.months_in_school_year)
+
+    def formula_2024(person, period, parameters):
```

Comment:
Could we do all of the 2024 formulas with bools in one formula as opposed to separating into two formulas. 

This will also give us the chance to compare the formula change over the same year by adjusting the bool parameter